bundle/phases/deploy.go

```diff
@@ -177,7 +177,12 @@ func Deploy(ctx context.Context, b *bundle.Bundle, outputHandler sync.OutputHand
 		deploy.StatePull(),
 		mutator.ValidateGitDetails(),
 		artifacts.CleanUp(),
+		// libraries.CheckForSameNameLibraries() needs to be run after we expand glob references so we
+		// know what are the actual library paths.
+		// libraries.ExpandGlobReferences() has to be run after the libraries are built and thus this
```

Comment:
Just curious, is this intentionally not part of the build step?

Reply:
Just logically it does not belong in build as it expands all type of libraries even the ones that might not be built by DABs